guides/common/modules/proc_synchronizing-templates-with-a-local-directory-using-the-api.adoc

```diff
@@ -28,20 +28,24 @@ organizations:
 ----
 
 .Procedure
-. Create the directory where templates are stored and apply appropriate permissions and SELinux context:
+. In `/var/lib/foreman`, create a directory for storing templates:
 +
 [subs="+quotes"]
 ----
-# mkdir -p _/usr/share/templates_dir/_
-# chown foreman _/usr/share/templates_dir/_
```

Comment:
This is still needed: if you create it as  root then Foreman can't write to it.

Reply:
Thanks, restored.